crates/llm-ls/src/error.rs

```diff
@@ -35,10 +35,12 @@ pub enum Error {
     OpenAI(crate::backend::OpenAIError),
     #[error("index out of bounds: {0}")]
     OutOfBoundIndexing(usize),
-    #[error("line out of bounds: {0}")]
-    OutOfBoundLine(usize),
+    #[error("line out of bounds: {0}..{1}")]
+    OutOfBoundLine(usize, usize),
     #[error("slice out of bounds: {0}..{1}")]
     OutOfBoundSlice(usize, usize),
+    #[error("range out of bounds: {0:?}")]
+    InvalidRange(Range),
```

Comment:
Everything is sorted by alphabetical order, can you move the error to its right place please? 😇

Reply:
Certainly! Apologies for any oversights and errors; I was rushing a bit too much...

Reply:
No worries, that's the point of reviews 🤗